src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowType/Index.cshtml

```diff
@@ -117,19 +118,27 @@
         $(".dropdown-menu .dropdown-item").filter(function() {
             return $(this).data("action");
         }).on("click", function () {
-            if ($(":checkbox[name='itemIds']:checked").length > 1) {
+            if ($(":checkbox[name='itemIds']:checked").length > 0) {
                 var $this = $(this);
-                confirmDialog({title: $this.data('title'), message: $this.data('message'), callback: function(r) {
-                    if (r) {
-                        $("[name='Options.BulkAction']").val($this.data("action"));
-                        $("[name='submit.BulkAction']").click();
-                    }
-                }});
+                switch ($this.data("action")){
+                case "Delete":
+                     confirmDialog({title: $this.data('title'), message: $this.data('message'), callback: function(r) {
+                        if (r) {
+                            $("[name='Options.BulkAction']").val($this.data("action"));
+                            $("[name='submit.BulkAction']").click();
+                        }
+                    }});
+                    break;
+                case "Export":
+                    $("[name='Options.BulkAction']").val($this.data("action"));
+                    $("[name='submit.BulkAction']").click();
+                    break;
+                }
             }
         });
 
         function displayActionsOrFilters() {
-            if ($(":checkbox[name='itemIds']:checked").length > 1) {
+            if ($(":checkbox[name='itemIds']:checked").length > 0) {
```

Comment:
Could be `$(":checkbox[name='itemIds']").is(":checked")`

Reply:
Because added the export button on the row, you revert this change